src/app/accounts/index.ts

```diff
@@ -35,6 +35,12 @@ export const getAccountFromUserId = async (
   return accounts.findByUserId(kratosUserId)
 }
 
+export const getAccountFromDeviceId = async (
+  deviceId: DeviceId,
+): Promise<Account | RepositoryError> => {
+  return accounts.findByUserId(deviceId as unknown as UserId)
```

Comment:
this casting doesn't sound right to me.

are we not using a kratos identity also for device account? is it a legacy function?

Reply:
yeah this a legacy function. I will delete it